### DISCUSSION TITLE
[fluff] Most hilarious code suggestion ever

### DISCUSSION THREAD

**Discussion by ffxsam** [community/community#8151] [Copilot Conversations]
This is Copilot basically saying "do it yourself!" 😄

<img width="482" alt="CleanShot 2021-11-26 at 15 10 29" src="https://user-images.githubusercontent.com/12532733/143652241-bf81c9b9-196c-4e35-8cd7-5e3f2d4fe529.png">

**Comment by Urethramancer** [comment]
I get those occasionally, and not even for very hard code. Sometimes it's part of a sequence of functions of the sort it handles fine in other cases. Sometimes it just stumbles randomly.

**Comment by alt-art** [comment]
This never happened to me but try using the next solution (`alt` + `[`)

**Reply by ffxsam** [reply]
Literally every synthesized suggestion was about the same. 😄

**Reply by alt-art** [reply]
Well, there will be a time when we will have to write by ourselves.

**Reply by ffxsam** [reply]
So far, Copilot has been great at saving me from writing obvious/tedious stuff. This thing is amazing.
